homeassistant/components/light/__init__.py

```diff
@@ -43,6 +43,9 @@
    A list containing three integers representing the xy color you want the
    light to be.
 
+ - ct_color
+   An INT in mireds represending the color temperature you want the light to be
+
```

Comment:
The value type for color_temp is a difficult topic.
There's mired, kelvin, 0-100% (i'm sure some devices will do this), perhaps more. Conversion is easy, but needs to become a utility function. 

Only 1 type can be used throughout HASS and must be converted in the platform for use with the actual device.
Range in color temp will probably differ between products, but i don't feel much for making the GUI slider min/max dynamic (and variables needed for that in the light component).

Making the 'main' value range 0-100 is easiest (for GUI integration), but lowers the quality of the color_temp support; color temp has more steps than 100.

So i left it in mired (native for hue), and tackle this when it becomes a problem.